integration-tests/Dockerfile

```diff
@@ -5,7 +5,7 @@ ENV LANG=en_US.UTF-8 \
     NOVENV=1
 
 RUN yum install -y epel-release &&\
-    yum install -y gcc which python34-pip python34-devel &&\
+    yum install -y gcc which python34-pip python34-devel centos-release-openshift-origin &&\
```

Comment:
Just FYI: We had some (can't remember exactly) problems with the latest `centos-release-openshift-origin`, so we [use centos-release-openshift-origin36 in deployment](https://github.com/fabric8-analytics/fabric8-analytics-deployment/blob/master/openshift/Dockerfile#L8)

Reply:
Updated :+1: